src/main/java/Event.java

```diff
@@ -0,0 +1,26 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+    private Date timeL;
+    private Date timeR;
```

Comment:
Good use of camelcase for naming variables, possible suggestion to use variable names timeStart/timeEnd for more information?